python/lsst/cp/pipe/ptc.py

```diff
@@ -159,6 +160,21 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Use bootstrap for the PTC fit parameters and errors?.",
         default=False,
     )
+    doPhotodiode = pexConfig.Field(
+        dtype=bool,
+        doc="Apply a correction based on the photodiode readings if available?",
+        default=True,
+    )
+    photodiodeDataPath = pexConfig.Field(
+        dtype=str,
+        doc="Gen2 only: path to locate the data photodiode data files.",
+        default=""
+    )
+    maxAduForLookupTableLinearizer = pexConfig.Field(
```

Comment:
I think there might be a merge issue here.  `maxAduForLookupTableLinearizer` was moved to `linearity.py`, and it looks like that change got lost here.

Reply:
Oh, great spot, I don't know how I missed that, thanks!